### DISCUSSION TITLE
1.19.2的工业 forge端

### DISCUSSION THREAD

**Discussion by dysslz** [IndustrialCraftMC/IndustrialCraft2#85] [【漏洞反馈】Bugs Report]
forge端的工业2 （1.19.2） 工业版本 industrialcraft-2-2.9.21 ex119-1.19.2-forge.jar 装上后 无法经入存档 是100%触发
fabric端的工业 可以正常进入

**Comment by dysslz** [comment]
1.19.2的工业fabric端 新加的TNT名字：Nuke Classic 右键放地上后游戏崩溃

**Comment by AbbottMc** [comment]
崩溃日志麻烦提供一下，我这边好排查问题

**Reply by dysslz** [reply]
ok

**Reply by dysslz** [reply]
[错误报告-fabric TNT.zip](https://github.com/IndustrialCraftMC/IndustrialCraft2/files/11543542/-fabric.TNT.zip)
[错误报告-forge无法进入存档的.zip](https://github.com/IndustrialCraftMC/IndustrialCraft2/files/11543543/-forge.zip)
gei给
